dev/reference/mysql-compatibility.md

```diff
@@ -149,3 +149,23 @@ TiDB supports **all of the SQL modes** from MySQL 5.7 with minor exceptions:
         - For MySQL 5.7: `OFF`
         - For MySQL 8.0: `ON`
     
+### Date and Time
+
+#### Named timezone
+
+TiDB supports named timezones such as `America/Los_Angeles` without having to load the [time zone information tables](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation) as in MySQL.
+
+Because they are built-in, named time zones in TiDB might behave slightly differently to MySQL, and cannot be modified. For example, in TiDB the names are case-sensitive [#8087](https://github.com/pingcap/tidb/issues/8087).
+
+> **Note:**
+>
+> TiKV calculates the time-related expressions received with its built-in named timezone calculation rules instead of the system named timezone rules. If these two kinds of named timezone rules do not match with each other, it might occur that the inserted time data cannot be read again. For example, if you install the tzdata 2018a named timezone rules, when you set the named timezone to Asia/Shanghai or to the server's timezone which is Asia/Shanghai, TiDB 3.0 RC.1 can be inserted to `1988-04-17 02:00:00`. But this record can not be read again by certain types of SQL statements. The reason is that the time does not exist in `Asia/Shanghai` named timezone according to the tzdata 2018i rules on which the TiKV 3.0 based. That is, the time does not exist because the DST shifts time forward by an hour.
```

Comment:
The translation of this paragraph has many problems. I cannot see the meaning conveyed in the Chinese version. Please check.

```suggestion
> TiKV calculates time-related expressions that can be pushed down to it. This calculation uses the built-in time zone rule and does not depend on the time zone rule installed in the system. If the time zone rule installed in the system does not match the version of the built-in time zone rules in TiKV, the time data that can be inserted cannot be read in a few cases.
>
> For example, if the tzdata 2018a time zone rule is installed in the system, the time `1988-04-17 02:00:00` can be inserted into TiDB 3.0 RC.1 when the time zone is set to Asia/Shanghai or the time zone is set to the local time zone and the local time zone is Asia/Shanghai. But this record can no longer be read out by specific types of SQL statements because this time does not exist in the Asia/Shanghai time zone according to the tzdata 2018i time zone rule used by TiKV 3.0 RC.1. Daylight saving time is one hour late.
```